src/components/CircleDetail/CircleDetail.tsx

```diff
@@ -0,0 +1,28 @@
+import '../Main/Main.css';
+
+interface IFiles {
+    files: number
+}
+
+function CircleDetail({files} : IFiles) {   
+    if (files < 3) {
+        return (
+            <div className='circles'>
+                <div className='circle floor'></div>
+                <div className='circle middle'></div>
+            </div>
+        )
+    } else {
+        return (
+            <div className='circles'>
+                <div className='circle floor'></div>
+                <div className='circle middle'></div>
+                <div className='circle withText ceil'>
+                    <p className='plusFiles'>+3</p>
+                </div>
+            </div>
+        )
+    }
```

Comment:
You should avoid leaving empty JSX components (`<div></div>`) and instead add then as self closed (`<div />`)
```suggestion
        return (
            <div className='circles'>
                <div className='circle floor'/>
                <div className='circle middle'/>
                {files < 3 && (
                  <div className='circle withText ceil'>
                    <p className='plusFiles'>+3</p>
                </div>
                )}
            </div>
        )
```

Reply:
True. I don't believe I added a linter to the project, but now I am aware of this convention. Thanks!